resource_booking/tests/test_backend.py

```diff
@@ -645,22 +645,22 @@ def test_videocall_location(self):
         with Form(rb) as rb_f:
             rb_f.start = "2021-03-01 08:00:00"
         self.assertEqual(rb.state, "scheduled")
-        self.assertEqual(rb.videocall_location, "Videocall Office 1")
-        self.assertEqual(rb.meeting_id.videocall_location, "Videocall Office 1")
+        self.assertTrue("Videocall Office 1" in rb.videocall_location)
```

Comment:
Use `assertIn` instead (in all occurrences). Anyway, I don't get why you are changing this.

Reply:
I have left `assertEqual`, although it is now necessary to specify http or https to avoid auto-defining with the base url.